src/components/AppRouter.jsx

```diff
@@ -1,24 +1,14 @@
 import { Route, Routes } from 'react-router-dom';
 import Home from '../pages/Home';
 import VideoDetails from '../pages/VideoDetails';
-import PropTypes from 'prop-types';
 
-const AppRouter = ({ loading, error, videoData }) => {
+const AppRouter = () => {
   return (
     <Routes>
-      <Route
-        path="/"
-        element={<Home loading={loading} error={error} videoData={videoData} />}
-      />
+      <Route path="/" element={<Home />} />
```

Comment:
Also remember that you can use this for routes when you don't need to pass anything
```
<Route path="/" component={Home} />
```